.evergreen/mongodl.py

```diff
@@ -0,0 +1,873 @@
+"""
```

Comment:
Might be better to make it "python3" instead of "python". And maybe with a fallback to one of the python or server toolchains:
```
C:/python/Python39/python.exe
/opt/mongodbtoolchain/v3/bin/python3
/opt/python/3.9/bin/python3
```

Reply:
@kevinAlbs I added a `--verison=latest` feature, although that may be different than what you're looking for since that will auto-update the moment a new `full.json` is published.

If the intent is to make it as easy as possible to update the pulled `csfle` version, would it not be at least as easy to change the `--version=` option used in a build script as it would be to update their copy of the drivers-evergreen-tools repo?

Reply:
Driver tests automatically install drivers-evergreen-tools@master so the advantage is that we can update drivers-evergreen-tools once and the change will be automatically applied to all driver tests.

Reply:
I like the `--version=latest` option. Language drivers can use that by default. In the rare scenario that a driver needs to pin to an older release, they could still do so in their respective repos.